### PR TITLE
ppc64 build cleanups and updated openblas for power

### PR DESCRIPTION
**Pull request by ViralBShah**
Check for both ppc64le and powerpc64le as the arch names consistently across the build. 

Use a newer openblas for power8.

### CODE REVIEW

File: deps/openblas.version [hunk 1]
```diff
@@ -1,2 +1,7 @@
+ifneq (,$(filter $(ARCH), powerpc64le ppc64le))
+OPENBLAS_BRANCH=develop
```
Comment (tkelman) [inline deps/openblas.version]: is there a tag for this?

Comment (ViralBShah) [inline deps/openblas.version]: Not yet, but I believe 0.2.19 is imminent soon - a day or two. Given that the rc4 may need some more time, let's wait and see if we can use a tagged version here.

Comment (tkelman) [inline deps/openblas.version]: you'd need to add checksums too

**Comment by ViralBShah** [comment]
This should be good to go now. OpenBLAS updated to v0.2.19 for Power.

**Comment by ViralBShah** [comment]
Build failures are mostly unrelated.

**Comment by ViralBShah** [comment]
@tkelman Can we get this one into rc4? Thanks.

**Comment by tkelman** [comment]
Yes. I'll add it. Don't need to ping me again, the label is enough.

**Pull request merged**